transformation/src/main/scala/org/broadinstitute/monster/hca/HcaPipelineBuilder.scala

```diff
@@ -543,6 +505,8 @@ object HcaPipelineBuilder extends PipelineBuilder[Args] {
           case None =>
             Option(SchemaValidationError(errPath, s"No schema found at $url"))
         }
+        a.foreach { err => err.log }
```

Comment:
Since you don't need `a` after this. you could write it without it. Also I think you can use an anonymous variable here:
```
        schemaOption match {
            ...
        }.foreach(_.log)
```